console/react/src/details/detailsTablePage.js

```diff
@@ -42,7 +42,7 @@ import { dataMap } from "../overview/entityData";
 import { dataMap as detailsDataMap, defaultData } from "./entityData";
 import Updated from "../common/updated";
 
-class DetailTablesPage extends React.Component {
+class DetailsTablePage extends React.Component {
```

Comment:
Just aligning the name with the rest of the app (I got confused by this before, I grepped for the class name ans so it nowhere used.)

Reply:
Good catch. Since the class name is the default export, it can be imported as any name. And indeed, it is imported as DetailsTablePage in layout.js. But for sanity sake, it is definitely better to have the export and import name match.